x/ccv/provider/keeper/proposal.go

```diff
@@ -159,14 +159,14 @@ func (k Keeper) StopConsumerChain(ctx sdk.Context, chainID string, lockUbd, clos
 	var vscIDs []uint64
 	if !lockUbd {
 		// iterate over the consumer chain's unbonding operation VSC ids
-		k.IterateOverUnbondingOpIndex(ctx, chainID, func(vscID uint64, ids []uint64) bool {
+		k.IterateOverUnbondingOpIndex(ctx, chainID, func(vscID uint64, ids []uint64) (stop bool) {
 			// iterate over the unbonding operations for the current VSC ID
 			var maturedIds []uint64
 			for _, id := range ids {
 				unbondingOp, found := k.GetUnbondingOp(ctx, id)
 				if !found {
 					err = fmt.Errorf("could not find UnbondingOp according to index - id: %d", id)
-					return false
+					return true // stop the iteration
```

Comment:
Also not relevant for this PR necessarily, but we might want to panic here? And/or consider if we should keep iterating if we don't panic

Reply:
We do return an error below, see https://github.com/cosmos/interchain-security/blob/f2e403a25f851678a0dd1b07fdaa7a8ca55abc38/x/ccv/provider/keeper/proposal.go#L195